### DISCUSSION TITLE
Issues while following starter documents

### DISCUSSION THREAD

**Discussion by markbSSL** [iPlug2/iPlug2#872] [Q&A]
Hey,
Im unsure if others have had any issues following the starter documents but I thought I would start a list so quirks us beginners may not understand can be explained and fixes can be shared.

1. PaceFixLogs
Ive begun with 02_Getting_Started_Windows in the wiki.
I opened the IPlugInstrument.sln file originally with visual studio 22 and tried to build the .app project and got a "PaceFixLogs" error. This went away when I opened the project in visual studio 19. However I can see (via git diff) that each IPlugInstrument-PLUGINFORMAT.vcxproj file has the line `<UsingTask TaskName="PaceFixLogs" AssemblyFile="$(PACE_FUSION_HOME)PaceFusionUi2013.dll" />` removed. What does this line do and do I need it in?

2. Building the vst3 app
I then changed the startup project to the vst3 version and ran it, however reaper couldnt open the vst3. The problem was due to the vst3 file IPlugInstrument.vst/Contents/ containing a x86-win and a x86_64-win folder. The x86-win folder contains the vst3 in a higher folder than the x86_64-win one and repear priorities vsts based on folder heirachy. I think this means it was trying to read the 32 bit vst3 which are being phased out and therefore an updated reaper would have issues with it? After I deleted the x86-win version, reaper could host the vst3 and everything worked fine!

3. Changing default DAW for debugging
I am a begginer with iPlug2, but I have been using JUCE for a little while and I use reaper for debugging vst3s there. This iPlug2 project seems to start a new reaper project when I begin debugging rather than allowing me to use my already setup test bench  something that I find annoying but I can work around Im sure by editing the common-win.props file? 
Anyway I wanted to change DAWs for this and used the select_host.py script to try and change it to Cubase, however the script couldnt find it! This is because I am running cubase 12 and the script is looking for cubase 11 - an easy fix to make to the script however I feel this should be configurable in the script. When you choose which DAW you want to use it could ask a version number too and substitute that into the string?

(I am new to github so please direct me to the right place for posting this if it is not here)

**Comment by olilarkin** [comment]
not used to looking at github discussions either, the forum or slack are more active

1) this line in the vcxproj is an annoying thing that is added by the AAX sdk IIRC. It can be removed, but when you try and build AAX it will come back.
2) I'll investigate, i think maybe we'll remove 32bit support soon anyway
3) You can completely customise that, and feel free to make a PR to add your desired feature to select_host.py
